crypto/rand/rand_lib.c

```diff
@@ -25,6 +25,45 @@ static CRYPTO_RWLOCK *rand_meth_lock;
 static const RAND_METHOD *default_RAND_meth;
 static CRYPTO_ONCE rand_init = CRYPTO_ONCE_STATIC_INIT;
 
+#ifdef OPENSSL_RAND_SEED_RDTSC
+/*
+ * Since we get some randomness from the low-order bits of the
+ * high-speec clock, it can help.  But don't return a status since
+ * it's not sufficient to indicate whether or not the seeding was
+ * done.
+ */
+void rand_rdtsc(void)
+{
+    unsigned char c;
+    int i;
+
+    for (i = 0; i < 10; i++) {
+        c = (unsigned char)(OPENSSL_rdtsc() & 0xFF);
```

Comment:
I actually did a test with a tight loop too, which gave me something like 0.02 which seems more realistic of what the TSC really contains.

I also tried with 100 Hz, which gives:
```
- Collision Estimate: p(max) = 0.0142759, min-entropy = 6.13028
- Markov Estimate (map 6 bits): p(max) = 4.29711e-219, min-entropy = 5.66718
```

Anyway, what you seem to be saying is that the time of sample itself contains some uncertainty, but might possible be under attacker control.